crates/storage/codecs/src/lib.rs

```diff
@@ -21,6 +21,8 @@ pub use reth_codecs_derive::*;
 
 use alloy_primitives::{Address, Bloom, Bytes, FixedBytes, U256};
 use bytes::{Buf, BufMut};
+#[cfg(feature = "optimism")]
+use op_alloy_consensus as _;
```

Comment:
why is this necessary?

Reply:
Maybe I missed something with the imports but otherwise it triggers the `unused_crate_dependencies` clippy (maybe a false positive). I have the impression that since `op_alloy_consensus` is only used in a submodule under the `optimism` cfg, it creates this weird warning...

However I thought I did things right in the `Cargo.toml` with an optional and a dep only for the `optimism` cfg..

Reply:
Imo this is similar to what we have here 

https://github.com/paradigmxyz/reth/blob/5df03fb3c3d70b3974ede5da7174bd9b9336ded0/crates/primitives/src/alloy_compat.rs#L11